src/foam/nanos/app/SMTPConfig.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.app',
```

Comment:
Move this into foam.nanos.notification.email, please.  Yes, I missed the Twillio, that should move, as well as the EmailConfig.  Later.